Framework/DataHandling/src/SaveNXcanSAS.cpp

```diff
@@ -722,26 +760,36 @@ void SaveNXcanSAS::init() {
   declareProperty(std::make_unique<Mantid::API::FileProperty>("Filename", "", API::FileProperty::Save, ".h5"),
                   "The name of the .h5 file to save");
 
-  std::vector<std::string> radiation_source{"Spallation Neutron Source",
-                                            "Pulsed Reactor Neutron Source",
-                                            "Reactor Neutron Source",
-                                            "Synchrotron X-ray Source",
-                                            "Pulsed Muon Source",
-                                            "Rotating Anode X-ray",
-                                            "Fixed Tube X-ray",
-                                            "neutron",
-                                            "x-ray",
-                                            "muon",
-                                            "electron"};
+  std::vector<std::string> radiationSourceOptions{"Spallation Neutron Source",
+                                                  "Pulsed Reactor Neutron Source",
+                                                  "Reactor Neutron Source",
+                                                  "Synchrotron X-ray Source",
+                                                  "Pulsed Muon Source",
+                                                  "Rotating Anode X-ray",
+                                                  "Fixed Tube X-ray",
+                                                  "neutron",
+                                                  "x-ray",
+                                                  "muon",
+                                                  "electron"};
   declareProperty("RadiationSource", "Spallation Neutron Source",
-                  std::make_shared<Kernel::StringListValidator>(radiation_source), "The type of radiation used.");
+                  std::make_shared<Kernel::StringListValidator>(radiationSourceOptions), "The type of radiation used.");
+  std::vector<std::string> const geometryOptions{Sample::CYLINDER, Sample::FLAT_PLATE, Sample::DISC};
+  declareProperty("Geometry", std::string(Sample::DISC), std::make_shared<Kernel::StringListValidator>(geometryOptions),
+                  "The geometry type of the collimation.");
+  declareProperty("SampleHeight", 0.0,
```

Comment:
I recall adding new parameters to an algorithm in the middle of the parameter list once and it was pointed out to me that this will break any scripts that are using positional arguments. Would it be better to add these new parameters to the end of the list? If moving them is an issue then I think we could also make sure that the parameter change is really clearly highlighted in the release notes.